pyproject.toml

```
@@ -66,7 +48,7 @@ exclude = [
     # TODO: Remove this in a follow-up where we fix __all__.
     "__init__.py",
 ]
-line-length = 88
+line-length = 79
 
```

```suggestion
[tool.ruff.format]
exclude = [
    "thirdparty",
    ".eggs",
    ".git",
    ".hg",
    ".mypy_cache",
    ".tox",
    ".venv",
    "_build",
    "buck-out",
    "build",
    "dist",
]
```

Porting the ignore list from the black config.